src/components/Footer/style.ts

```diff
@@ -10,7 +9,8 @@ export const FooterContainer = styled.footer`
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: ${theme.color.gray[1]};
+  background-color: ${props => props.theme.color.gray[1]};
```

Comment:
```suggestion
  background-color: ${({ theme }) => theme.color.gray[1]};
```

props를 받아올 때, 이런 식으로 줄일 수 있겠네요.

Reply:
그렇네요! 코드를 간소화 하면서 안정적으로  동작할 수 있도록 노력하겠습니다!